bookmarks.html

```diff
@@ -98,31 +98,31 @@ <h2 class="pdb-10">Question 2</h2>
       </main>
       <nav class="app-grid__fixed-btn nav">
         <ul class="nav--display-content">
-          <li class="no-bul nav__icon nav__icon:hover nav__icon:active">
+          <li class="no-bul nav__icon">
```

Comment:
im not sure if you really need a class for the `li ` ...if you never need a bullet in your app i wuld tag the `li` in the base.css